core/sql/exp/ExpLOB.h

```diff
@@ -264,11 +265,11 @@ class ExpLOBoper : public ex_clause {
 
   // Generates LOB handle that is stored in the SQL row.
   // LOB handle max len:  512 bytes
-  // <flags><LOBnum><objectUid><LOBlen><descKey><descTS><chunkNum><schNameLen><schName>
-  // <--4--><--4---><----8----><---8--><---8---><--8---><----2---><---2------><--vc--->
+  // <flags><LOBType><LOBnum><objectUid><LOBlen><descKey><descTS><chunkNum><schNameLen><schName>
+  // <--4--><--4----><--4---><----8----><---8--><---8---><--8---><----2---><---2------><--vc--->
   static void genLOBhandle(Int64 uid, 
```

Comment:
Is this change upward-compatible? What happens if this code is installed on an existing instance with tables that already have LOB data?